<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QMainWindow" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_choose">
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit_file1">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_chooseFile1">
          <property name="text">
           <string>choose file1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_file2">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_chooseFile2">
          <property name="text">
           <string>choose file2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_handle">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_reset">
          <property name="text">
           <string>reset</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_process">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>process</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton_chooseFile1</tabstop>
  <tabstop>pushButton_chooseFile2</tabstop>
  <tabstop>pushButton_process</tabstop>
  <tabstop>lineEdit_file1</tabstop>
  <tabstop>lineEdit_file2</tabstop>
  <tabstop>pushButton_reset</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_file1</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWidget</receiver>
   <slot>isAllFilesChosen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_file2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWidget</receiver>
   <slot>isAllFilesChosen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>isAllFilesChosen()</slot>
 </slots>
</ui>
